app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java

```diff
@@ -132,6 +141,11 @@ public String getCurrentUsername() {
         return mPref.getString("username", "null");
     }
 
+    public String getCurrentSchoolId()
+    {
+        return mPref.getString("currentSchool", "null");
```

Comment:
Same thing here, see all other getters for string values:
https://github.com/schul-cloud/schulcloud-mobile-android/blob/01178ff91828e9b5d76f20ecb4b1cc6a0e79f605/app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java#L129
https://github.com/schul-cloud/schulcloud-mobile-android/blob/01178ff91828e9b5d76f20ecb4b1cc6a0e79f605/app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java#L133
https://github.com/schul-cloud/schulcloud-mobile-android/blob/01178ff91828e9b5d76f20ecb4b1cc6a0e79f605/app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java#L137
https://github.com/schul-cloud/schulcloud-mobile-android/blob/01178ff91828e9b5d76f20ecb4b1cc6a0e79f605/app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java#L141
https://github.com/schul-cloud/schulcloud-mobile-android/blob/01178ff91828e9b5d76f20ecb4b1cc6a0e79f605/app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java#L150
https://github.com/schul-cloud/schulcloud-mobile-android/blob/01178ff91828e9b5d76f20ecb4b1cc6a0e79f605/app/src/main/java/org/schulcloud/mobile/data/local/PreferencesHelper.java#L158